tests/runner.js

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env node
-require( 'babel-core/register' );
+require( 'babel-core/register' )( {
+	ignore: /\/node_modules\/(?!(@automattic|dops-components)\/)/
```

Comment:
Ah good catch. wp-calypso is no longer a dependency of Jetpack so it may have been left there by mistake in webpack's config file